packages/indexer-agent/package.json

```diff
@@ -10,7 +10,7 @@
   "scripts": {
     "format": "prettier --write src/*.ts src/**/*.ts src/**/**/*.ts",
     "lint": "eslint . --ext .ts,.tsx",
-    "compile": "./node_modules/.bin/tsc",
+    "compile": "tsc",
```

Comment:
Will this work under all platforms wherein the path doesn't always get set up correctly?

Reply:
Good question.  I don't have a definitive answer, but found that this makes it work with NPM whereas the node_modules/bin location execution we were using before did not.